platforms/cloudflare/crypto.md

```diff
@@ -0,0 +1,60 @@
+# Crypto
+
+The crypto section is used for everything relating to SSL certificates, TLS and HTTPS rewriting.
+
+Click on the Crypto tab to get started: 
+
+![Crypto tab](images/crypto/crypto-tab.jpg "Crypto tab")
```

Comment:
How does this work? Isn't this just an image?

Reply:
It shows the tab to click on while in the dashboard. It is obvious but there's no reason why it couldn't be there.

Reply:
Ah, maybe I misread it then. I hadn't seen a mention of a dashboard in any of the documentation 📄